cli/wallet/multisig.go

```diff
@@ -89,3 +94,13 @@ func signStoredTransaction(ctx *cli.Context) error {
 	fmt.Fprintln(ctx.App.Writer, tx.Hash().StringLE())
 	return nil
 }
+
+func dumpTransaction(ctx *cli.Context, tx *transaction.Transaction) {
+	fmt.Fprintf(ctx.App.Writer, "Hash:\t%s\n", tx.Hash().StringLE())
+	fmt.Fprintf(ctx.App.Writer, "SystemFee:\t%s\n", fixedn.Fixed8(tx.SystemFee))
+	fmt.Fprintf(ctx.App.Writer, "NetworkFee:\t%s\n", fixedn.Fixed8(tx.NetworkFee))
```

Comment:
Don't we want to print `ValidUntilBlock` value? It's easy to overlook and sign an outdated transaction.